observations/forward_operators/obs_def_abs_humidity_mod.f90

```diff
@@ -0,0 +1,186 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+!
+! $Id$
+
+! BEGIN DART PREPROCESS KIND LIST
+!MPD_ABSOLUTE_HUMIDITY,    QTY_ABSOLUTE_HUMIDITY
+! END DART PREPROCESS KIND LIST
+
+! BEGIN DART PREPROCESS USE OF SPECIAL OBS_DEF MODULE
+!   use obs_def_abs_humidity_mod, only : get_expected_absolute_humidity
+! END DART PREPROCESS USE OF SPECIAL OBS_DEF MODULE
+
+! BEGIN DART PREPROCESS GET_EXPECTED_OBS_FROM_DEF
+!         case(MPD_ABSOLUTE_HUMIDITY)
+!            call get_expected_absolute_humidity(state_handle, ens_size, location, expected_obs, istatus)
+! END DART PREPROCESS GET_EXPECTED_OBS_FROM_DEF
+
+! BEGIN DART PREPROCESS READ_OBS_DEF
+!         case(MPD_ABSOLUTE_HUMIDITY)
+!            continue
+! END DART PREPROCESS READ_OBS_DEF
+
+! BEGIN DART PREPROCESS WRITE_OBS_DEF
+!         case(MPD_ABSOLUTE_HUMIDITY)
+!            continue
+! END DART PREPROCESS WRITE_OBS_DEF
+
+! BEGIN DART PREPROCESS INTERACTIVE_OBS_DEF
+!         case(MPD_ABSOLUTE_HUMIDITY)
+!            continue
+! END DART PREPROCESS INTERACTIVE_OBS_DEF
+
+! BEGIN DART PREPROCESS MODULE CODE
+module obs_def_abs_humidity_mod
+
+use        types_mod, only : r8, missing_r8, gas_constant, gas_constant_v
+use    utilities_mod, only : register_module, error_handler, E_ERR, E_MSG, E_ALLMSG
+use     location_mod, only : location_type, set_location, get_location, write_location, &
+                             read_location, is_vertical
+use  assim_model_mod, only : interpolate
+use     obs_kind_mod, only : QTY_TEMPERATURE, QTY_PRESSURE, QTY_VAPOR_MIXING_RATIO
+
+use ensemble_manager_mod,  only : ensemble_type
+use obs_def_utilities_mod, only : track_status
+
+
+implicit none
+private
+
+public :: get_expected_absolute_humidity
+
+! version controlled file description for error handling, do not edit
+character(len=256), parameter :: source   = &
+   "$URL$"
+character(len=32 ), parameter :: revision = "$Revision$"
+character(len=128), parameter :: revdate  = "$Date$"
```

Comment:
These can be rewritten:
```
character(len=*), parameter :: source   = 'obs_def_abs_humidity_mod.f90'
character(len=*), parameter :: revision = ''
character(len=*), parameter :: revdate  = ''
```
since the svn keywords were stripped out when files got converted to git, we need to manually do this now.
We are abandoning the revision/revdate, so those don't need to be filled in.